package.json

```diff
@@ -32,7 +32,7 @@
     "eslint-config-standard": "^10.2.1",
     "eslint-config-semistandard": "^11.0.0",
     "eslint-plugin-header": "^1.0.0",
-    "request": "~2.81.0",
+    "request": "^2.83.0",
     "browser-request": "~0.3.0",
     "debug": "^3.0.0"
   },
```

Comment:
Add an engine tag for at least Node 4.x. Otherwise newer versions of request are likely to make this fail.